### ISSUE TITLE
[bug] blocking SIWX error in the latest version of reown

### ISSUE THREAD

**Issue by NeOMakinG** [reown-com/appkit#3273]
### Link to minimal reproducible example

https://www.dailymeme.io/ but it's fixed now

### Summary

You recently refactored SIWE to SIWX which cause our website signin to crash, as it's included inside a minor version the update has been done automatically by npm

<img width="560" alt="image" src="https://github.com/user-attachments/assets/181f416d-c5dc-4e4e-9620-3dc5e9895616">

As it's a minor version, the package has been updated automatically and completely killed the signin of our website in production, meaning that absolutely everyone might be impacted

### List of related npm package versions

We fixed it by removing the `^` from this list:

```
    "@reown/appkit": "1.4.1",
    "@reown/appkit-adapter-wagmi": "1.4.1",
    "@reown/appkit-siwe": "1.4.1",
```

So we don't get the `1.5.0` version automatically

**Comment by rtomas** [comment]
hi @NeOMakinG, I hope you are doing good. 

We cannot reproduce this issue. 
Would you mind sharing with me more context and your AppKit Configuration ?

We add some better error msg to siwx to identify easily the root cause ...
https://github.com/reown-com/appkit/pull/3276

**Comment by NeOMakinG** [comment]
Our repository is private and it's super hard to share everything but here is the exact details on what did happen:

- On localhost, we didn't have any issue as node_modules were cached.
- We are using these dependencies (notice the ^ character)
```
    "@reown/appkit": "^1.4.1",
    "@reown/appkit-adapter-wagmi": "^1.4.1",
    "@reown/appkit-siwe": "^1.4.1",
```
- We delivered our app to production but in the meantime you pushed the version `1.5.0`
- The new app has been built using the latest `1.5.0` version because we didn't pin the versions 1.4.1
- The error message did shown and we couldn't sign to any wallet

What I don't get is:

Why do we have a SIWX error message while we use the old appkit-siwe package alongside appkit and appkit-adapter-wagmi and not the appkit-siwx package

I think there is a breaking change where our old configuration is not compatible anymore with the minor version 1.5.0, but this is a breaking change and considering semver it shouldn't be introduced in a minor version...

**Comment by zoruka** [comment]
Hey @NeOMakinG! Thank you for reporting the issue! 

After the version `1.5.0`, internally AppKit is now treating SIWE as SIWX that's why we got this error message. But the interfaces for old configuration has not changed and they are expected to be working the same as before, resulting in non-breaking changes, that's why we pushed it in a minor version.

We have some example apps from which we have tested this same structure you have shared for [next](https://github.com/reown-com/web-examples/tree/main/dapps/appkit-siwe/next) and [react](https://github.com/reown-com/web-examples/tree/main/dapps/appkit-siwe/react) and they were working as expected. We are not sure exactly why this have happened there, is there any other error in the console happening before the log you have shared?
